server/scripts/seedDB.js

```diff
@@ -0,0 +1,90 @@
+const mongoose = require("mongoose");
+const usersDB = require("../models/user");
+const chefsDB = require("../models/chef");
+const mealsDB = require("../models/meal");
+
+// This file empties all the Users, Chefs, and Meals collections and inserts the seeds.
+
+mongoose.connect("mongodb://localhost/chefsmenu", { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.set('useCreateIndex', true);
+
+// Seed for users collection
+const userSeed = 
+  {
+    first_name: "Megan",
+    last_name: "Gilpin",
```

Comment:
Need to match this to the new camelCase in the model.
Can you also add a brief explanation of how to use this to the readme?

Reply:
Thanks for camelCase call out and will for sure add to the readme.